crates/router/src/core/payments/operations/payment_create.rs

```diff
@@ -1207,6 +1206,7 @@ impl PaymentCreate {
                 attempt_id,
                 status,
                 currency,
+                order_tax_amount: request.order_tax_amount,
```

Comment:
can you remove this and test once, if everything is working? we should get the order amount in attempt in confirm only

Reply:
order_tax_amount is a field in PaymentAttempt, so this is included here in PaymentAttemptNew struct also